include/superior_mysqlpp/prepared_statements/prepared_statement_base.hpp

```diff
@@ -192,45 +254,71 @@ namespace SuperiorMySqlpp
                 return ok;
             }
 
-            /**
-             * TODO: Statement.fetchColumn is void and can throw
```

Comment:
Why this comment is deleted?

Reply:
This comment is deleted because this function was in fact not working at all - and is rewritten along with proper documentation in following pull request #63.